Neos.ContentRepository.Core/Classes/Projection/Projections.php

```diff
@@ -93,4 +93,8 @@ public function getIterator(): \Traversable
     {
         yield from $this->projections;
     }
+    public function isEmpty(): bool
+    {
+        return count($this->projections) === 0;
```

Comment:
Mini nitpick, this is _slightly_ faster and what we use in other places
```suggestion
        return $this->projections === [];
```

Reply:
I never really liked the readability of this (to me) but will change it.

Reply:
for me it's the other way around, but it's probably a matter of habit, so nevermind!